internal/service/kafka/serverless_cluster.go

```diff
@@ -184,6 +189,17 @@ func resourceServerlessClusterRead(ctx context.Context, d *schema.ResourceData,
 		return sdkdiag.AppendErrorf(diags, "setting vpc_config: %s", err)
 	}
 
+	output, err := findBootstrapBrokersByARN(ctx, conn, d.Id())
+
+	switch {
+	case errs.IsA[*types.ForbiddenException](err):
+		d.Set("bootstrap_brokers_sasl_iam", nil)
```

Comment:
Can we open a follow up issue to remove this handling in a future major version? (e.g. `kafka:GetBootstrapBrokers` will be explicitly required).

Reply:
https://github.com/hashicorp/terraform-provider-aws/issues/42555